### ISSUE TITLE
svg inline plot mess up jupyter notebook

### ISSUE THREAD

**Issue by yech1990** [matplotlib/matplotlib#14552]
When render matplotlib figure by svg backend within jupyter  notebook. The whole webpage will be messed up.  The header bar of the page will occupy the  whole page. The only method to show the code and result is size down the web page. 

---

Any kind of plot can trigger this bug. This is a simply demo:

```python
import matplotlib.pyplot as plt
%config InlineBackend.figure_format = "svg"

plt.plot([1,2, 3])
plt.show()
```

The `png` backend works fine.

![pngback](https://user-images.githubusercontent.com/5415510/59487531-96af4280-8eaf-11e9-87b1-196cf3cbf67c.png)

The `svg` backend messed up everything.

![svgerror](https://user-images.githubusercontent.com/5415510/59487532-98790600-8eaf-11e9-94f8-a311780dbab5.png)

- It can be reproduced in the latest  matplotlib version(3.1.0), but not in the previous one(3.0.3)
- notebook version is `5.7.8`

**Comment by timhoffm** [comment]
Whithout checking, I assume that this is leaking CSS. Maybe related: #14283?

**Comment by yech1990** [comment]
@timhoffm  seem that this patch can fix this problem.

But the master version conflicts with seaborn package functions. eg `#sns.set(context="paper", style="ticks")`

**Comment by ImportanceOfBeingErnest** [comment]
@yech1990 Could you go into detail about the problem you face with current master and seaborn functions?

**Comment by lukelbd** [comment]
I have the same problem. Here's before printing the figure:

<img width="1232" alt="svg-backend1" src="https://user-images.githubusercontent.com/19657652/61089522-a5c6d900-a3f8-11e9-998a-ffbdf148172d.png">

And here's after:

<img width="1188" alt="svg-backend2" src="https://user-images.githubusercontent.com/19657652/61089523-a5c6d900-a3f8-11e9-81de-97d1bc9163c8.png">

If I delete the inline figure output, everything reverts back to the first example.

Here is the result of `jupyter --version`:

```
jupyter core     : 4.5.0
jupyter-notebook : 5.7.8
qtconsole        : 4.5.1
ipython          : 7.6.1
ipykernel        : 5.1.1
jupyter client   : 5.2.4
jupyter lab      : not installed
nbconvert        : 5.5.0
ipywidgets       : 7.4.2
nbformat         : 4.4.0
traitlets        : 4.3.2
```

**Comment by ImportanceOfBeingErnest** [comment]
@lukelbd This is fixed in matplotlib 3.1.1 (via https://github.com/matplotlib/matplotlib/pull/14403).   
(Unfortunately, 3.1.1 introduced another [serious bug](https://github.com/matplotlib/matplotlib/issues/14675) preventing seaborn heatmaps to render correctly; so up to you to decide if you want to revert to 3.0.3, or update to 3.1.1)

**Comment by QuLogic** [comment]
Closing as fixed then.
